tests/bypass-quic-enabled/test.yaml

```diff
@@ -0,0 +1,18 @@
+requires:
+    min-version: 9
+
+pcap: ../quic-v2/input.pcap
+
+args:
+- --set app-layer.protocols.quic.encryption-handling=bypass
+- --set stream.reassembly.depth=1MB
```

Comment:
stream settings shouldn't affect udp protocols. Did you need to set them?

Reply:
No, this was meant to me more explicit that the bypass comes from quic and not from stream depth

Removing it